packages/msw/package.json

```diff
@@ -35,10 +35,15 @@
         "react": "*",
         "react-dom": "*"
     },
+    "peerDependenciesMeta": {
```

Comment:
Do we still want to keep it as optional even if it didnt work?

Reply:
Oh sorry, the error was commit from missing `compilerOptions` paths in the `tsconfig` file. It does work! :)